src/components/pages/ItemList.js

```diff
@@ -0,0 +1,221 @@
+          })
+        // console.log(res.data)
+      )
+      .catch(err => console.log(err));
+
+    Axios.get("http://localhost:5000/items/")
+      .then(
+        res =>
+          this.setState({
+            available_items: res.data
+          })
+        // console.log(res.data)
+      )
+      .catch(err => console.log(err));
+
+    Axios.get("http://localhost:5000/items/")
+      .then(
+        res =>
+          this.setState({
+            remain_items: res.data
+          })
+        // console.log(res.data)
+      )
+      .catch(err => console.log(err));
+  }
+
+  getItems() {
+    // console.log("**********************");
+    // console.log(this.state.itemlist);
+    // console.log(this.state.available_items);
+    // console.log("_____________________");
+    // console.log(this.state.available_items2);
+
+    const item = [];
+    for (var i = 0; i < this.state.itemlist.length; i++) {
+      //console.log(this.state.itemlist[i]._id);
+      if (this.state.itemlist[i]._id === this.state.orderid) {
+        this.state.items = this.state.itemlist[i].items;
+        // console.log(this.state.itemlist[i].items);
+      }
+    }
+    // console.log(this.state.items);
+    // console.log(this.state.items.length);
+    // console.log("&&&&&&&&&&&&&&&&&&&&&&");
+
+    const qty = [];
+    for (var i = 0; i < this.state.items.length; i++) {
+      for (var j = 0; j < this.state.available_items.length; j++) {
+        if (this.state.items[i].Itemid === this.state.available_items[j]._id) {
+          //   console.log(this.state.items[i].Itemid);
+          //   console.log(this.state.items[i].Qty);
+          qty[i] = this.state.available_items[j].Qty - this.state.items[i].Qty;
+          this.state.available_items[j].Qty = qty[i];
+          //   console.log(qty[i]);
+
+          //   console.log(this.state.available_items[j]);
+
+          //   this.state.ordered_items[i].itemid = this.state.available_items[
+          //     j
+          //   ]._id;
+          //   console.log(this.state.ordered_items[i].Itemid);
+
+          this.state.ordered_items[i] = this.state.available_items[j];
+
+          this.state.ordered_items[i].Qty = this.state.items[i].Qty;
+        }
+      }
+    }
+
+    this.state.subtotal = 0;
+    //this.setState({ subtotal: 0 });
+    console.log(this.state.ordered_items);
+    for (var i = 0; i < this.state.ordered_items.length; i++) {
+      this.state.subtotal =
+        this.state.subtotal +
+        this.state.ordered_items[i].Qty * this.state.ordered_items[i].unitprice;
+    }
+    // console.log(this.state.subtotal);
+    //this.setState({ subtotal: this.total });
+  }
+
+  onClick(e, id) {
+    console.log(id);
+    console.log(e.target.value);
+    console.log(this.state.available_items);
+    var ids = "";
+    for (var i = 0; i < this.state.available_items.length; i++) {
+      if (e.target.value === this.state.available_items[i].Itemname) {
+        console.log("****");
```

Comment:
Please get rid of these kind logs, if we don't get these right at this moment, this can become a habbit.

Reply:
Okay.